micromamba/src/activate.cpp

```diff
@@ -28,23 +28,35 @@ set_activate_command(CLI::App* subcom)
         [&]()
         {
             std::string guessed_shell = guess_shell();
-            std::string shell_hook_command = "", shell_hook = "";
+            std::string shell_hook_command = "", shell_hook = "", shell_hook_dev_mode_cmd = "",
+                        shell_hook_dev_mode = "";
 
             if (guessed_shell == "powershell")
                 shell_hook_command
                     = "micromamba.exe shell hook -s powershell | Out-String | Invoke-Expression";
             else
+            {
                 shell_hook_command
                     = "eval \"$(micromamba shell hook --shell=" + guessed_shell + ")\"";
 
+                shell_hook_dev_mode_cmd = "eval \"$(./micromamba/micromamba shell hook --shell="
```

Comment:
Won't this path change depending on user? It's not uncommon to put it under `build/`

Reply:
In my experience, I also don't need to run the hook again. Since the hook already uses the `MAMBA_EXE` variable, I can set `export MAMBA_EXE="${PWD}/build/micromamba/micromamba`. Or perhaps I'm missing something.

Reply:
> Won't this path change depending on user? It's not uncommon to put it under `build/`

If you are testing your built micromamba, I assumed you're in your build directory and so the path would be the same. But we could also make it `./build/micromamba/micromamba`, maybe that would be clearer.